packages/@webex/plugin-meetings/src/meeting/util.ts

```diff
@@ -150,7 +150,9 @@ const MeetingUtil = {
         ipVersion: MeetingUtil.getIpVersion(meeting.getWebexObject()),
       })
       .then((res) => {
-        // @ts-ignore
+        const parsed = MeetingUtil.parseLocusJoin(res);
+
+        meeting.locusInfo.initialSetup(parsed.locus);
```

Comment:
I'm not sure if we need to do a second initial setup if it's already been done. Inside Locus info, there are a couple of interesting functions that sound like something that could be useful here, like `updateLocusInfo` and `onDeltaLocus`. Have you tried using them?

Reply:
initialSetup is called on every locusevent handled, not just first initialization. 
From what I can see, updateLocusInfo and onDeltaLocus are used more like internal updates and initialSetup as external update.

Reply:
As we discussed, I want to double-check if these two logics are not repeating each other. It seems that we are making two calls to `initialSetup` for locusInfo in the join meeting flow. If this is the case, it would be better to consolidate it into one.